modules/renter/worker.go

```diff
@@ -22,27 +22,21 @@ type worker struct {
 	hostPubKey types.SiaPublicKey
 	renter     *Renter
 
-	// Channels that inform the worker of kill signals and of new work.
-	downloadChan         chan downloadWork // higher priority than all uploads
-	killChan             chan struct{}     // highest priority
-	priorityDownloadChan chan downloadWork // higher priority than downloads (used for user-initiated downloads)
-	uploadChan           chan struct{}     // lowest priority
-
-	// Operation failure statistics for the worker.
-	downloadRecentFailure     time.Time // Only modified by the primary download loop.
-	uploadRecentFailure       time.Time // Only modified by primary repair loop.
-	uploadConsecutiveFailures int
-
-	// Two lists of chunks that relate to worker upload tasks. The first list is
-	// the set of chunks that the worker hasn't examined yet. The second list is
-	// the list of chunks that the worker examined, but was unable to process
-	// because other workers had taken on all of the work already. This list is
-	// maintained in case any of the other workers fail - this worker will be
-	// able to pick up the slack.
-	mu                sync.Mutex
-	standbyChunks     []*unfinishedChunk
-	terminated        bool
-	unprocessedChunks []*unfinishedChunk
+	// Download variables.
+	downloadChan          chan downloadWork // higher priority than all uploads
+	downloadRecentFailure time.Time
+	priorityDownloadChan  chan downloadWork // higher priority than downloads (used for user-initiated downloads)
+
+	// Uploading variables.
+	unprocessedChunks         []*unfinishedChunk // Yet unprocessed work items.
+	uploadChan                chan struct{}      // Notifications of new work.
+	uploadConsecutiveFailures int                // How many times in a row uploading has failed.
+	uploadRecentFailure       time.Time          // How recent was the last failure?
+	uploadTerminated          bool               // Have we stopped uploading?
+
+	// Utilities.
+	killChan chan struct{} // Worker will shut down if a signal is sent down this channel.
+	mu       sync.Mutex
```

Comment:
Though I re-arranged a bunch of stuff, I also deleted the 'standbyChunks' field.